CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -15,6 +15,10 @@
 <p>The new Find View offers an updated interface that opens by default in a separate window.</p> 
 <p>The functionality remains the same with the added option of "Find Next" and "Find Previous" execution options.</p> 
 
+== 2023-06-23 Markdown support in the Notes View
+<p>The notes will now render markdown text. You can now italicise, bold and strikethrough text. Headings can be added to the note as well as lists and sub lists.</p>
+<p>The new note pop-up window will have a preview tab where you can see how your markdown text will be rendered before adding it as a note.</p>
```

Comment:
Minor update required to the description since both Creating and Editing notes will be done in a popup, and Markdown is optional.